setup.py

```diff
@@ -50,11 +52,23 @@
 ):
     # This is the system install of Python on OS X, install to `/etc`.
     target_prefix = '/'
-completetion_dest = os.path.join(target_prefix, 'etc/bash_completion.d')
+completion_dest = os.path.join(target_prefix, 'etc/bash_completion.d')
 data_files.append((
-    completetion_dest,
+    completion_dest,
     ['completion/catkin_tools-completion.bash']))
 
+
+class PermissiveInstall(install):
+    def run(self):
+        install.run(self)
+        if os.name == 'posix':
+            for file in self.get_outputs():
+                # all installed files should be readable for anybody
+                mode = ((os.stat(file)[ST_MODE]) | 0444) & 07777
```

Comment:
This line is not valid Python 3 syntax. You have to write octal number in the following style: `0o1234`